### PR TITLE
feat: Include aliases toBeSameAs and toBe for toBeSame

### PR DESCRIPTION
**Pull request by CristhianMotoche**
# Changes

Include `toBeSameAs` and `toBe` alias for `toBeSame`.

*NOTE:* My code formatter modified some files. Let me know if it's okay to keep it that way, otherwise I can revert that to keep only the necessary changes.﻿

Solves #108 

---
As part of [Stack Builders Inc.](https://www.stackbuilders.com/) Hacktoberfest 2023 event.

### CODE REVIEW

File: packages/core/test/lib/Assertion.test.ts [hunk 1]
```diff
@@ -472,6 +471,28 @@ describe("[Unit] Assertion.test.ts", () => {
     });
   });
 
+  describe(".toBeSameAs", () => {
```
Comment (CristhianMotoche) [inline packages/core/test/lib/Assertion.test.ts:474]: These are the changes that matter in this file for this PR.

File: packages/core/src/lib/Assertion.ts [hunk 2]
```diff
@@ -440,6 +444,48 @@ export class Assertion<T> {
     });
   }
 
+  /**
```
Comment (CristhianMotoche) [inline packages/core/src/lib/Assertion.ts:447]: These are the changes that matter in this file for this PR.

**Review by JoseLion — COMMENTED** [review]
The relevant changes look great! Do you think we can revert the formatter changes, though? 🙂

**Comment by CristhianMotoche** [comment]
Hi @JoseLion I reverted the code formatter changes. Let me know if looks better now.

**Review by JoseLion — APPROVED** [review]
Looks great! Thanks for your contribution @CristhianMotoche 🎉

**Pull request merged**